IS-1219: implement getAccount API in SDK (#58)
icon-project/icon-sdk-python: README.md

# Returns an estimated step value
estimate_step = icon_service.estimate_step(transaction)
```

## Get Account

> Get raw data about account in State Database for debug.

Reviewer: ```suggestion
Get the raw data of account in State Database.
```
